code/modules/tgui_panel/telemetry.dm

```diff
@@ -226,6 +226,8 @@
 				return "!"
 			if(alert_low)
 				return "?"
+			if(alert_dev)
+				return "dev"
 			return ""
```

Comment:
the real fix is just to make this something clickable yet small, like an empty box